fn/routes.go

```diff
@@ -278,7 +278,8 @@ func envAsHeader(req *http.Request, selectedEnv []string) {
 }
 
 func (a *routesCmd) create(c *cli.Context) error {
-	if c.Args().Get(0) == "" || c.Args().Get(1) == "" || c.Args().Get(2) == "" {
+	// todo: @pedro , why aren't you just checking the length here?
+	if c.Args().Get(0) == "" || c.Args().Get(1) == "" {
 		return errors.New("error: routes creation takes at least three arguments: app name, route path and image name")
```

Comment:
Error message doesn't match logic.  Logic is checking app name and route path only.  But error message also mentions image name as required.